techlibs/xilinx/abc9_xc7.box

```diff
@@ -48,12 +72,13 @@ CARRY4 4 1 10 8
 #        input/output and the entire bus has been
 #        moved there overriding the otherwise
 #        alphabetical ordering)
-CARRY4_COUT 8 1 10 5
-482 -   -   -   -   223 -   -   -   222
-598 407 -   -   -   400 205 -   -   334
-584 556 537 -   -   523 558 226 -   239
-642 615 596 438 -   582 618 330 227 313
-580 526 507 398 385 508 528 378 380 114
+CARRY4_COUT 8 0 10 5
```

Comment:
Why did you change this at all?  This should be identical to before?

Reply:
Specifically why did `CARRY4_COUT 8 1` become `CARRY4_COUT 8 1`?

Reply:
I believe as well this change is not correct. I still need to understand why we get an error in case the CARRY4_OUT is a whitebox.

The error I am getting is the following:
```
K = 8. Memory (bytes): Truth =    0. Cut =   64. Obj =  144. Set =  672. CutMin = no
Node =     901.  Ch =   112.  Total mem =    0.23 MB. Peak cut mem =    0.04 MB.
P:  Del = 3843.00.  Ar =    1348.0.  Edge =     1274.  Cut =    14144.  T =     0.00 sec
P:  Del = 3825.00.  Ar =    1385.0.  Edge =     1340.  Cut =    13812.  T =     0.00 sec
P:  Del = 3825.00.  Ar =    1071.0.  Edge =     1107.  Cut =    24151.  T =     0.01 sec
F:  Del = 3816.00.  Ar =     735.0.  Edge =      936.  Cut =    17855.  T =     0.01 sec
A:  Del = 3816.00.  Ar =     666.0.  Edge =      851.  Cut =    16942.  T =     0.01 sec
A:  Del = 3816.00.  Ar =     636.0.  Edge =      821.  Cut =    17224.  T =     0.01 sec
Total time =     0.03 sec
+ &mfs
yosys-abc: src/aig/gia/giaMfs.c:388: Gia_Man_t *Gia_ManInsertMfs(Gia_Man_t *, Sfm_Ntk_t *, int): Assertion `iLitNew >= 0' failed.
Aborted (core dumped)
```

I will look into the synth.tcl script to check if there is some parameters missing that may solve the issue.